test/timing_load_creds.c

```diff
@@ -108,14 +123,20 @@ int main(int ac, char **av)
     /* Read input file. */
     if (av[0] == NULL)
         usage();
-    if (stat(av[0], &sb) < 0)
-        perror(av[0]), exit(1);
+    if (stat(av[0], &sb) < 0) {
+        perror(av[0]);
+        exit(1);
```

Comment:
Strictly, these should all be `exit(EXIT_FAILURE);`

Reply:
Right - changed all of them.